diff --git a/data.py b/data.py
index 4a9d342..95fee0a 100644
--- a/data.py
+++ b/data.py
@@ -420,7 +420,7 @@ def decode_jpeg(jpeg_data):
             img = cv2.imdecode(img_array, cv2.IMREAD_COLOR)
             
             if img is None:
-                with open("./bad_Data.jpg", "wb") as w:
+                with open("/tmp/bad_Data.jpg", "wb") as w:
                     w.write(jpeg_data)
                     quit()
                 raise Exception("OpenCV decoding failed")
@@ -512,8 +512,8 @@ def read_capture_file(filename, exclude_after=0, exclude_before=0):
     #print("Detecting corrupted BSB frames...", flush=True)
     last_was_safe = False
     ibl_db = {}
-    if os.path.exists("./ibl_db_%d.pt" % crc):
-        ibl_db = torch.load("./ibl_db_%d.pt" % crc, weights_only=False)
+    if os.path.exists("/tmp/ibl_db_%d.pt" % crc):
+        ibl_db = torch.load("/tmp/ibl_db_%d.pt" % crc, weights_only=False)
     with open(filename, 'rb') as f:
         progress = range(total_frames)
         for e in progress:
@@ -596,8 +596,8 @@ def read_capture_file(filename, exclude_after=0, exclude_before=0):
                                             routine_widen, routine_squint, routine_dilate, routine_state)
             
             #print(f"Read frame: Pitch={routine_pitch}, Yaw={routine_yaw}, sizeRight={len(image_right_data)}, sizeLeft={len(image_left_data)}, timeData={timestamp}, timeLeft={video_timestamp_left}, timeRight={video_timestamp_right}")
-    if not os.path.exists("./ibl_db_%d.pt" % crc):
-        torch.save(ibl_db, "./ibl_db_%d.pt" % crc)
+    if not os.path.exists("/tmp/ibl_db_%d.pt" % crc):
+        torch.save(ibl_db, "/tmp/ibl_db_%d.pt" % crc)
 
     # Convert to sorted lists for processing
     left_frames = sorted([(ts, img) for ts, img in all_eye_frames_left.items()])
diff --git a/trainer_distsampler.py b/trainer_distsampler.py
index 2305d72..c30eb2b 100644
--- a/trainer_distsampler.py
+++ b/trainer_distsampler.py
@@ -420,7 +420,7 @@ def decode_jpeg(jpeg_data):
             img = cv2.imdecode(img_array, cv2.IMREAD_COLOR)
             
             if img is None:
-                with open("./bad_Data.jpg", "wb") as w:
+                with open("/tmp/bad_Data.jpg", "wb") as w:
                     w.write(jpeg_data)
                     quit()
                 raise Exception("OpenCV decoding failed")
@@ -512,8 +512,8 @@ def read_capture_file(filename, exclude_after=0, exclude_before=0):
     #print("Detecting corrupted BSB frames...", flush=True)
     last_was_safe = False
     ibl_db = {}
-    if os.path.exists("./ibl_db_%d.pt" % crc):
-        ibl_db = torch.load("./ibl_db_%d.pt" % crc, weights_only=False)
+    if os.path.exists("/tmp/ibl_db_%d.pt" % crc):
+        ibl_db = torch.load("/tmp/ibl_db_%d.pt" % crc, weights_only=False)
     with open(filename, 'rb') as f:
         progress = range(total_frames)
         for e in progress:
@@ -596,8 +596,8 @@ def read_capture_file(filename, exclude_after=0, exclude_before=0):
                                             routine_widen, routine_squint, routine_dilate, routine_state)
             
             #print(f"Read frame: Pitch={routine_pitch}, Yaw={routine_yaw}, sizeRight={len(image_right_data)}, sizeLeft={len(image_left_data)}, timeData={timestamp}, timeLeft={video_timestamp_left}, timeRight={video_timestamp_right}")
-    if not os.path.exists("./ibl_db_%d.pt" % crc):
-        torch.save(ibl_db, "./ibl_db_%d.pt" % crc)
+    if not os.path.exists("/tmp/ibl_db_%d.pt" % crc):
+        torch.save(ibl_db, "/tmp/ibl_db_%d.pt" % crc)
 
     # Convert to sorted lists for processing
     left_frames = sorted([(ts, img) for ts, img in all_eye_frames_left.items()])
